sympy/core/exprtools.py

```diff
@@ -1418,7 +1418,7 @@ def _pemexpand(expr):
                         ok = hit = True
                         l = b**e
                         il = b**-e
-                        for j, a in enumerate(args):
+                        for j in range(len(args)):
```

Comment:
Could this be

    for a in args:
        a[1][0] = il*a[1][0]

Reply:
Maybe. I am not sure about the object behavior, i.e. is the `a` that is changed still the same object that is in `args`? If it is, then it should be possible.

(A few lines down there is:
```
for i, a in enumerate(args):
    args[i][1][0] = i1*a[1][0]
```
or something similar, which also is an option. So whoever wrote this code seems to have the same knowledge gap as I have...)

Reply:
Nice, @jksuom ! Of course...if you are changing a list element you can do so via its reference:
```python
>>> args = [[1,2],[3,4]]
>>> for a in args:
...     a[0]*=-1
>>> args
[[-1, 2], [-3, 4]]
```

Reply:
> whoever wrote this code

I am probably the guilty party and this occurs in some other places, too.